src/assets/config.yaml

```diff
@@ -1,6 +1,5 @@
 credentials:
   usernames:
-
 cookie:
   expiry_days: 30
   key: random_signature_key # Must be string
```

Comment:
do we need to do something with this?

Reply:
Yes, added the link to the documentation